website/content/docs/templates/hcl_templates/functions/datetime/formatdate.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-page_title: formatdate - Functions - Configuration Language
-description: The formatdate function converts a timestamp into a different time format.
+page_title: formatdate function reference
+description: The `formatdate` function converts a timestamp into a different time format. Learn how to use `formatdate` functions in Packer templates.
```

Comment:
```suggestion
description: The `formatdate` function converts a timestamp into a specified time format. Learn how to use `formatdate` functions in Packer templates.
```

What do you think of something like this? IMO we should specify that it takes in a format string with the timestamp.